airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/oauth.py

```diff
@@ -75,11 +75,11 @@ def get_grant_type(self) -> str:
     def get_token_expiry_date(self) -> pendulum.DateTime:
         return self._token_expiry_date
 
-    def set_token_expiry_date(self, initial_time: pendulum.DateTime, value: Union[str, int]):
+    def set_token_expiry_date(self, value: Union[str, int]):
```

Comment:
should we rename this method because it's not setting the expiry date to `value`?

Reply:
I think the name of this method is accurate -- it's expected to do whatever processing necessary to store a timestamp of when the access token expires